comp/logs/agent/agentimpl/agent.go

```diff
@@ -240,9 +238,26 @@ func (a *logAgent) startPipeline() {
 		a.pipelineProvider,
 		a.diagnosticMessageReceiver,
 		a.launchers,
-		a.schedulers,
 	)
 	starter.Start()
+
+	if !sds.ShouldBlockCollectionUntilSDSConfiguration(a.config) {
+		a.startSchedulers()
+		a.log.Info("logs-agent started")
+	} else {
+		a.log.Info("logs-agent ready, schedulers not started")
```

Comment:
Would it make sense to also log why? And perhaps highlight this condition (ready, not started) in `agent status`?